apis_core/core/static/css/core.css

```diff
@@ -1,11 +1,7 @@
+/* ACDH-CH logo in site footer */
```

Comment:
```suggestion
/* Logo in site footer */
```

Reply:
Thanks, I hadn't considered it's meant to be a place holder. D'oh.

I'd actually like to make that clearer  – i.e. change it to sth. like "project logo", "app logo", "site logo". What do you think?

Reply:
I've updated the wording. Can remove it again, but think it doesn't hurt to be specific about it.